hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -871,6 +874,7 @@ public static boolean scmBootstrap(OzoneConfiguration conf)
               getScmAddress(scmhaNodeDetails, conf), false);
         }
         scmStorageConfig.setPrimaryScmNodeId(scmInfo.getScmId());
+        scmStorageConfig.setSCMHAFlag(true);
```

Comment:
Do we need to set this flag only when HA enabled to avoid user mistakes of running bootStrap on non-HA cluster?

Reply:
BootStrap will fail if HA is not enabled right upfront. It won't execute this code at all.

Reply:
Yes.